Sources/Penny/Handlers/InteractionHandler.swift

```diff
@@ -451,6 +452,171 @@ private extension InteractionHandler {
                 return """
                 Renamed a FAQ from '\(oldName)' to '\(name)':
 
+                \(value)
+                """
+            }
+        case let .autoFaqs(autoFaqsMode):
+            switch autoFaqsMode {
+            case .add:
+                let expression = try modal.components
+                    .requireComponent(customId: "expression")
+                    .requireTextInput()
+                    .value.requireValue()
+                let newValue = try modal.components
+                    .requireComponent(customId: "value")
+                    .requireTextInput()
+                    .value.requireValue()
+
+                if expression.contains("\n") {
+                    let expNoNewLines = expression.replacingOccurrences(of: "\n", with: " ")
```

Comment:
```suggestion
                    let expNoNewLines = expression.replacing("\n", with: " ")
```

Reply:
hmm if that's one of the new regex functions, it'll probably cause problems with "static-stdlib" flag 🤔

Reply:
I see there are regex variants too ... I'm not sure if it won't just infer to use the regex variant.
I'm going to give it a try.